docs/troubleshooting.asciidoc

```diff
@@ -14,6 +14,30 @@ further to identify the offending code.
 Of course, there are errors that might have been caused by the agent itself and we appreciate it if
 you <<get-in-touch, report them back to us>>.
 
+[float]
+[[cross-origin-script-error]]
+=== Some errors in the application only show `script error`
+In some cases when you look at the details of an error, the only information you can see is the message `Script error`.
+This happens when an error originates from a JavaScript file served from a different origin than the page's origin.
```

Comment:
Would it be helpful if we include this [reference](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror#notes)?

Reply:
Yes, good idea! 🎉